### DISCUSSION TITLE
alignment to e-clavis

### DISCUSSION THREAD

**Discussion by PietroLiuzzo** [SChAth/ScAthiop#1806] [Ideas]
align data to https://www.nasscal.com/e-clavis-christian-apocrypha/
URI?
identifiers?
contact curators
import their alignments.

**Comment by thea-m** [comment]
It would be also nice if, in addition to our aligment in BM, Manuscripta Apocryphorum could point to BM in "other online databases" for the three manuscripts currently covered by both projects: 
https://www.nasscal.com/manuscripta-apocryphorum/london-british-library-or-605/ 
https://www.nasscal.com/manuscripta-apocryphorum/london-british-library-or-609/
https://www.nasscal.com/manuscripta-apocryphorum/london-british-library-or-685/

Have you been in contact with this project?

**Comment by PietroLiuzzo** [comment]
I am not in contact and I am unlikely to cover this  enhancement for the release.

**Comment by thea-m** [comment]
@MarcinKrawczuk would also like to point to some of their entries in the bibliography of work records he's editing. What would be the best way to refer to, for instance, https://www.nasscal.com/e-clavis-christian-apocrypha/encomium-on-the-four-bodiless-creatures-by-pseudo-john-chrysostom/?
Create a zotero item for the e-clavis (what would it be, "Computerprogramm"?) and then use the ECCA reference  or create a zotero entry for the single entry (Document? blog post?)?

**Comment by PietroLiuzzo** [comment]
I think the item type in Zotero would be web page. simply importing from browser will result in that type, which is not in the lists for the creation of new items in Zotero. I have just added one automatically, which will certainly need editing.
```xml
<bibl>
   <ptr target="bm:nasscal"/>
   <citedRange unit="item">ECCA 685</citedRange>
  <ref target="https://www.nasscal.com/e-clavis-christian-apocrypha/encomium-on-the-four-bodiless-creatures-by-pseudo-john-chrysostom/"/>
</bibl>
```
If this is going to be used in the clavis bibliography (which will need to be updated to handle that) it should look like this in that context, more or less (not sure about CPG1 and I do not know what CPC is)

```xml
<listBibl type="clavis">
   <bibl type="CPG"><ptr target="bm:CPG1"/><citedRange unit="item">5150.11</citedRange></bibl>
   <bibl type="ECCA"><ptr target="bm:nasscal"/><citedRange unit="item">685</citedRange></bibl>
</listBibl>
```

**Comment by PietroLiuzzo** [comment]
The information on "citing this resource" in the linked page is IMHO not very helpful, either to those willing to cite (as us) as it omits the identifier or to the ones reading that citation (whatever the style). The same as above can be done trusting extraction features of Zotero plugin, but will not be able to extract the "compiler" so, I think linking to the general website and pointing to a part of it with a link spelled out is better. the general record should include the editors https://www.nasscal.com/e-clavis-christian-apocrypha/

**Comment by MarcinKrawczuk** [comment]
I wasn't focused on the ECCA identifier (actually I wasn't aware that it is used in the BM), rather I wanted to refer to some of the information in the entry. Having said this, I will follow @PietroLiuzzo 's advice.
